services/src/main/scala/cromwell/pipeline/service/GitLabProjectVersioning.scala

```diff
@@ -46,7 +47,28 @@ class GitLabProjectVersioning(httpClient: HttpClient, config: GitLabConfig)
   override def getFileTree(project: Project, version: Option[Version])(
     implicit ec: ExecutionContext
   ): AsyncResult[List[String]] = ???
+
   override def getFile(project: Project, path: Path, version: Option[Version])(
     implicit ec: ExecutionContext
-  ): AsyncResult[String] = ???
+  ): AsyncResult[String] = {
+    val projectId: String = project.projectId.value
```

Comment:
You will remove this line after reading the next comment.

Reply:
Well, I don't think so..